src/data.js

```diff
@@ -70,6 +70,30 @@ module.exports = [
       'Blogger',
     ],
   },
+  {
+    name: 'David Smooke',
+    description: 'founder/ceo @hackernoon, leading the product team.',
+    url: 'https://www.davidsmooke.net/',
```

Comment:
Thanks, @davidsmooke. Will you update your site to have a uses page?
`url: 'https://www.davidsmooke.net/'`, needs to follow the format of

`use|uses|using|setup|environment` at the end.

After you fix that then we can get this merged in!

Reply:
Hey @BlakeCampbells! just looked through a couple examples and made: https://www.davidsmooke.net/uses does that work?

Reply:
That works @davidsmooke!